src/overlay_15_0238AE6C.c

```diff
@@ -2,8 +2,46 @@
 
 extern unkStruct_ov15_0238AE6C* OVERLAY15_UNKNOWN_POINTER__NA_238B180;
 
-extern void ov15_0238A140(void);
-extern void ov15_0238AD78(u32);
+extern u16 ov15_0238A140(void);
+extern void ov15_0238AD78(u8);
+extern void Debug_Print0(const char* fmt);
+extern void MemFree(void*);
+
+//TODO change these array literals into proper string representation and pass to Debug_Print0
+
+// "mes not close\n" with null termination and padding
+static const char ov15_0238B11C[] = {
+    0x6D, 0x65, 0x73, 0x20, // "mes "
```

Comment:
I got this to match with string literals by moving the consts just before `ov15_0238AE98`. For some reason if you use string literals and leave them at the top of the file, `ov15_0238B150` and `ov15_0238B13C` switch positions in the compiled ROM.

```
s32 ov15_0238AE88(void)
{
    ov15_0238A140();
    return 1;
}

const char ov15_0238B11C[] = "mes not close\n";
const char ov15_0238B12C[] = "sub not close\n";
const char ov15_0238B13C[] = "select not close\n";
const char ov15_0238B150[] = "Input not close\n";

void ov15_0238AE98(void)
{
```